modal/image.py

```diff
@@ -1162,6 +1181,24 @@ def dockerfile_commands(
         if not cmds:
             return self
 
+        if context_mount is not None:
```

Comment:
Might want to add the `ignore` argument here as well if people want manual inline ignore patterns without external files, as a replacement for context_mount. And the pending deprecation could point people towards that argument

Reply:
(or dockerignore obviously)

Reply:
Wait, does dockerfile_commands support dockerignore with this? We only call find_dockerignore_file in from_dockerfile right? 🤔 
No biggie if it doesn't - as long as we support `ignore` for now

Reply:
added to `dockerfile_commands` as well -- currently both methods will try to find a dockerignore file unless a custom ignore argument is passed